apps/etterlatte-saksbehandling-ui/client/src/components/behandling/attestering/handinger/attesterYtelse.tsx

```diff
@@ -22,8 +22,12 @@ export const AttesterYtelse = ({ behandling, kommentar }: { behandling: IDetalje
     apiAttesterVedtak(
       { behandlingId: behandling.id, kommentar },
       () => navigate(`/person/${behandling.søker?.foedselsnummer}`),
-      () => {
-        setError(`Ukjent feil oppsto ved attestering av vedtaket... Prøv igjen.`)
+      (error) => {
+        if (error.code === 'ATTESTANT_OG_SAKSBEHANDLER_ER_SAMME_PERSON') {
```

Comment:
Kor mykje av error-objektet frå backend får vi med over her? Dvs. kan vi bruke `detail` direkte?

Reply:
Vi får med alt, så `status`, `detail`, `code` og `meta` er alle tilgjengelig (om definert da). Du ser fallback'en til `detail` lengre nede. Tanken er egentlig at `detail` er en mer generell feilmelding som beskriver hva som går galt, code gir deg en mer programmatisk måte å hente ut hvilken feil det er, og meta kan inneholde ting som er spesifikt til akkurat denne feilen

Reply:
Ja, sant. Gir meining! Trur vi mange stadar da kan gjera det du her har gjort som fallback direkte, og da sleppe å ha denne mappinga både i frontend og backend. Gitt at vi skriv gode nok detail-meldingar i exception-klassa, sjølvsagt.

Reply:
Ja, det er nok det beste som standard, `code` er mest nyttig hvis man vil ha en spesifikk oppførsel for feil